docusaurus-plugin-moonwave/src/components/LuaType.js

```diff
@@ -1,11 +1,19 @@
-import React from "react"
+import Link from "@docusaurus/Link"
+import React, { useContext } from "react"
+import { TypeLinksContext } from "./LuaClass"
 import styles from "./styles.module.css"
 import { Op } from "./Syntax"
 
 const isPunc = (char) => !!char.match(/[\{\}<>\-\|]/)
 const isWhitespace = (char) => !!char.match(/\s/)
 const isAtom = (char) => !isWhitespace(char) && !isPunc(char)
 
+function isValidUrl(urlString) {
+  const urlPattern =
+    /^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)$/gm
+  return urlPattern.test(urlString)
+}
+
```

Comment:
We don't really need to check if something is a valid URL, only if it's internal or external. Simply checking if it starts with `/https?:\/\//` is enough.